README.md

```diff
@@ -27,7 +27,7 @@ terraform {
 }
 
 provider "onepassword" {
-  url = "http://localhost:8080"
+  service_account_token = "<1Password service account token>"
```

Comment:
Are we sure this is sufficient in terms of ensuring users will provide the service account token in a secure way? 🤔

Reply:
The idea of having `service_accouunt_token` here is to copy/paste the snippet and run it.

More details on how to use more secure way to setting env vars are in [Terraform Registry docs](docs/index.md) and [examples](examples/README.md)